Include signal.h

Fixes several signal set related errors:
src/basic/copy.c:92:19: error: implicit declaration of function 'sigemptyset' [-Werror=implicit-function-declaration]
src/basic/copy.c:93:19: error: implicit declaration of function 'sigaddset' [-Werror=implicit-function-declaration]
src/basic/copy.c:93:34: error: 'SIGINT' undeclared (first use in this function)
src/basic/copy.c:95:13: error: implicit declaration of function 'sigtimedwait' [-Werror=implicit-function-declaration]

Upstream-Status: Pending

Signed-off-by: Scott Murray <scott.murray@konsulko.com>
---
 src/basic/copy.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/basic/copy.c b/src/basic/copy.c
index ca311e021e..3cf7fc1697 100644
--- a/src/basic/copy.c
+++ b/src/basic/copy.c
@@ -12,6 +12,7 @@
 #include <sys/xattr.h>
 #include <time.h>
 #include <unistd.h>
+#include <signal.h>
 
 #include "alloc-util.h"
 #include "btrfs-util.h"
